backend/contributions/views.py

```
@@ -1723,44 +1769,36 @@ def get_queryset(self):
+                F('target_points') - F('awarded_amount'),
                 Value(0),
                 output_field=IntegerField(),
             )
         )
 
-    def _get_locked_state(self, contribution_id):
-        state = self.get_queryset().select_for_update(of=('self',)).get(
-            contribution_id=contribution_id,
-        )
-        contribution = state.contribution
+    def _get_locked_state(self, pk):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_get_locked_state`

(ANN202)